### ISSUE TITLE
nestpy v1.5.1 not available in `pip` 

### ISSUE THREAD

**Issue by JoranAngevaare** [NESTCollaboration/nestpy#83]
Nestpy `1.5.1` is not available for `python >= 3.8`. I don't know why, but I can highly recommend gitgub actions for [pipy uploads](https://github.com/NESTCollaboration/nestpy/pull/80#issuecomment-1029785530), as far as I've seen, it's very easy to configure and reliable.

```
(base) joran@DESKTOP-F4PI41P:~$ pip install nestpy==1.5.1
ERROR: Could not find a version that satisfies the requirement nestpy==1.5.1 (from versions: 1.0.1, 1.0.2, 1.0.3, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.2.1, 1.3.0, 1.3.3, 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.4.7, 1.4.8, 1.4.11, 1.5.0)
ERROR: No matching distribution found for nestpy==1.5.1
```

**Comment by ershockley** [comment]
We are seeing the same issue also for v1.5.2. @sophiafarrell @tunnell do you know what's going on?

**Comment by sophiafarrell** [comment]
Thanks for letting me know, @ershockley. My hunch is it has to do with Travis builds failing on linux only due to some docker image things that were not known to me.  @JoranAngevaare has generously agreed to help out in trying to fix this today since this has become more urgent.

**Comment by sophiafarrell** [comment]
Fixed, see #86 and thank you @JoranAngevaare for your help on this.
